### ISSUE TITLE
FM Go: disappearing icons in the new archival entity layout

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#3012]
@luizaogs :

> Restarted the iPad, only FM Go20 is open right now (Bridge is not open). 
> 
> I think I got it, or at least one way to cause it: if I search for a term that I know is not in any description and use the "next" key in the keyboard to do the search, this doesn't happen. It happens every single time I search for a term not in any records and click on the header, rather than using the next key, to get the search going.
> 
> So: clicking on "next" works fine. Clicking outside of the keyboard to run the search leads to the issue.
> 

_Originally posted by @luizaogs in https://github.com/arch-kiosk/arch-kiosk-office/issues/3007#issuecomment-2526438603_

**Comment by urapadmin** [comment]
@luizaogs :
> Still only with FM 20 open.

Huh, I guess I forgot to note this above but I also see a difference in screens. When using "next" I get the no records message against a white screen. When clicking elsewhere (eg the header) I see the message in the FM layout, and that's when I get the weird blank buttons showing up after I click on "aha..."

(When I say header, by the way, I don't mean the refresh button. That also gives me the message on a blank screen but then everything looks normal once I click on "aha")

**Comment by urapadmin** [comment]
That's what it looks like for @lbestock:

https://github.com/arch-kiosk/arch-kiosk-office/issues/3007#issuecomment-2527892011

**Comment by luizaogs** [comment]
Just for the record, since I have now updated the iPad: I get the same exact thing happening as described above in both FM 20 and 21 in iOS 18+.

**Comment by urapadmin** [comment]
I suspect it is a timing issue or something like that. I'll make a version with more room to breathe next.

**Comment by urapadmin** [comment]
does this ONLY happen when using the search term or does it happen with the starts / ends, too?

**Comment by luizaogs** [comment]
Only with search, and only if I'm searching for a term that leads to no records.

**Comment by urapadmin** [comment]
### recording app 16.7.2
- I have changed a few lines. Does it still happen?

**Comment by luizaogs** [comment]
No, but now using the refresh button gives me the no records message twice.

**Comment by urapadmin** [comment]
but only when you changed the search term, right? So you leave that field, which triggers filtering, then click the button and that triggers filtering again. Is that accurate?

Or do you get it twice whenever you click that button?

**Comment by luizaogs** [comment]
No. I get it twice. I click on aha, the screen flashes, and I get another no records warning.

**Comment by urapadmin** [comment]
and that happens again if you simply click again on the button?

**Comment by luizaogs** [comment]
You mean the refresh button?

**Comment by urapadmin** [comment]
yes

**Comment by luizaogs** [comment]
No, the second time doesn't do it it seems. At least not if I don't change the search term.

**Comment by urapadmin** [comment]
okay. Then yes, it is the succession of two things each of which triggers the filtering. Hm. I am not sure what to do there. There is no need to hit the refresh button when changing a word but I suppose the proximity of button and search field suggests it. I'll think about it. But I am glad the other more worrisome phenomenon vanished.

**Comment by urapadmin** [comment]
### recording app 16.8
- I hope this works now. Neither should you be able to produce the no records message twice in quick succession nor, I hope, should you still suffer from disappearing icons.
